servicetalk-dns-discovery-netty/src/main/java/io/servicetalk/dns/discovery/netty/DefaultDnsClient.java

```diff
@@ -222,6 +226,10 @@ final class DefaultDnsClient implements DnsClient {
             builder.nameServerProvider(toNettyType(dnsServerAddressStreamProvider));
         }
         resolver = builder.build();
+        this.resolutionTimeoutMillis = resolutionTimeout != null ? resolutionTimeout.toMillis() :
+                // Default value is chosen based on a combination of default "timeout" and "attempts" options of
+                // /etc/resolv.conf: https://man7.org/linux/man-pages/man5/resolv.conf.5.html
+                resolver.queryTimeoutMillis() * 2;
```

Comment:
Open for alternative suggestions. Originally, I planned to use `resolver.queryTimeoutMillis() * resolver.maxQueriesPerResolve()`, but Nett's default values are:
```
queryTimeout = 5 sec
maxQueriesPerResolve = 16
```
which results in 80 sec and seems unreasonably high. 

Decided to go with x2 because https://man7.org/linux/man-pages/man5/resolv.conf.5.html because the linux default for attempts (maxQueriesPerResolve) is 2.

Reply:
Fwiw, this seems reasonable to me.

Reply:
Seems reasonable to me as well 👍